src/components/Chat/MessageAttachments.tsx

```diff
@@ -26,7 +26,7 @@ export const MessageAttachments = ({ attachments, isInner }: Props) => {
   }
 
   return isUnderSection && !isInner ? (
-    <div className="rounded border border-gray-400 bg-gray-300  dark:border-gray-700 dark:bg-gray-900">
+    <div className="rounded border border-primary bg-layer-1">
```

Comment:
```suggestion
    <div className="rounded border border-secondary bg-layer-1">
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/bff70eac-29e0-4dd1-abaa-2748bea9dd6b)